elks/arch/i86/drivers/char/console-bios.c

```diff
@@ -19,6 +19,7 @@
 #include <linuxmt/chqueue.h>
 #include <linuxmt/ntty.h>
 #include <linuxmt/kd.h>
+#include <linuxmt/homebrew.h>
```

Comment:
I would suggest including `linuxmt/homebrew.h` in config.h in the `#ifdef CONFIG_HW_HOMEBREW` section. In this way, we won't need to add a homebrew.h include in every file that happens to require the SETUP_xxx values.